arm-web/2016-08-01/swagger/WebApps.json

```diff
@@ -10405,6 +10629,39 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/syncfunctiontriggers": {
+      "post": {
+        "tags": [
+          "WebApps"
+        ],
+        "summary": "this is to allow calling via powershell and ARM template",
+        "description": "this is to allow calling via powershell and ARM template",
+        "operationId": "WebApps_SyncFunctionTriggers",
```

Comment:
Please provide x-ms-examples section according to https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/x-ms-examples.md.